openmetadata-ui/src/main/resources/ui/src/components/Modals/KillIngestionPipelineModal/KillIngestionPipelineModal.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Modal } from 'antd';
+import { AxiosError } from 'axios';
+import React, { FC, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { killIngestionPipelineById } from '../../../axiosAPIs/ingestionPipelineAPI';
+import { showErrorToast, showSuccessToast } from '../../../utils/ToastUtils';
+
+interface KillIngestionModalProps {
+  pipelineId: string;
+  pipelinName: string;
+  isModalOpen: boolean;
+  onClose: () => void;
+}
+
+const KillIngestionModal: FC<KillIngestionModalProps> = ({
+  pipelineId,
+  pipelinName,
+  isModalOpen,
+  onClose,
+}) => {
+  const history = useHistory();
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const handleConfirm = async () => {
+    setIsLoading(true);
+    try {
+      const response = await killIngestionPipelineById(pipelineId);
+      const status = response.status;
+      if (status === 200) {
+        onClose();
+        showSuccessToast(
+          `Successfully killed running workflows for ${pipelinName}.`
+        );
+        // reload the page after successful api response
+        history.go(0);
```

Comment:
Instead of reloading the page, can we fetch the API response again and re-render the component with the updated pipeline status? That should be much faster than a reload right?

Reply:
Thanks, @vivekratnavel, I have removed the reload logic and added logic to fetch updated pipeline status.